neo/Prompt/Commands/Send.py

```diff
@@ -14,85 +15,155 @@
         if assetId is None:
             print("Asset id not found")
             return False
-
+        if type(assetId) is NEP5Token:
+            print('Sendmany does not support NEP5 tokens')
+            return False
+        address_to = prompt("Address to: ")
         scripthash_to = lookup_addr_str(wallet, address_to)
         if scripthash_to is None:
             print("invalid address")
             return False
+        amount = prompt("Amount to send: ")
+        f8amount = get_asset_amount(amount, assetId)
+        if f8amount is False:
+            print("invalid amount")
+            return False
+        if float(amount) == 0:
+            print("amount cannot be 0")
+            return False
+        tx_output = TransactionOutput(AssetId=assetId, Value=f8amount, script_hash=scripthash_to)
+        output.append(tx_output)
+        contract_tx = ContractTransaction(outputs=output)
 
         scripthash_from = None
 
         if from_address is not None:
             scripthash_from = lookup_addr_str(wallet, from_address)
+            if scripthash_from is None:
+                print("invalid address")
+                return False
 
-        # if this is a token, we will use a different
-        # transfer mechanism
-        if type(assetId) is NEP5Token:
-            return do_token_transfer(assetId, wallet, from_address, address_to, amount_from_string(assetId, amount), prompt_passwd=prompt_password, tx_attributes=user_tx_attributes)
+        scripthash_change = None
 
-        f8amount = Fixed8.TryParse(amount, require_positive=True)
-        if f8amount is None:
-            print("invalid amount format")
-            return False
-
-        if type(assetId) is UInt256 and f8amount.value % pow(10, 8 - Blockchain.Default().GetAssetState(assetId.ToBytes()).Precision) != 0:
-            print("incorrect amount precision")
-            return False
+        if change_address is not None:
+            scripthash_change = lookup_addr_str(wallet, change_address)
+            if scripthash_change is None:
+                print("invalid address")
+                return False
 
         fee = Fixed8.Zero()
         if priority_fee is not None:
             fee = priority_fee
+            if fee is False:
+                print("invalid fee")
+                return False
 
         print("sending with fee: %s " % fee.ToString())
+    return [contract_tx, scripthash_from, scripthash_change, fee, owners, user_tx_attributes]
 
-        output = TransactionOutput(AssetId=assetId, Value=f8amount, script_hash=scripthash_to)
-        tx = ContractTransaction(outputs=[output])
 
-        ttx = wallet.MakeTransaction(tx=tx,
-                                     change_address=None,
-                                     fee=fee,
-                                     from_addr=scripthash_from)
+def process_transaction(prompter, wallet, contract_tx, scripthash_from=None, scripthash_change=None, fee=None, owners=None, user_tx_attributes=None):
```

Comment:
can you please remove the `prompter` argument. It's not used. Same goes for all other functions in the `Send` file, they have `prompter` argument that's not used